src/main/java/org/primefaces/component/avatar/Avatar.java

```diff
@@ -34,5 +34,6 @@
     public static final String SIZE_LARGE_CLASS = "ui-avatar-lg";
     public static final String SIZE_XLARGE_CLASS = "ui-avatar-xl";
     public static final String SIZE_TEXT_CLASS = "ui-avatar-text";
+    public static final String DYNAMIC_TEXT_CLASS = "ui-avatar-dynamic";
```

Comment:
DYNAMIC_TEXT_CLASS or DYNAMIC_COLOR_CLASS?

Reply:
Updated to DYNAMIC_COLOR_CLASS